### PR TITLE
Set empty country, province, and organization subject fields to null in X.509 certificate

### PR DESCRIPTION
**Pull request by t4niwa**
# Description

This update modifies the behavior of X.509 certificate issuance to ensure that the Country, Province, and Organization fields in the Subject are set to null when they are empty strings.

## Assignees
- [x] `Assignees` is set

## Type of changes
- [x] Apply one or more `labels` of the following that fits:
  - `bug`: Bug fix
  - `dependencies`: Dependency upgrades
  - `documentation`: Documentation changes
  - `enhancement`: New Feature
  - `good first issue`: First contribution
  - `logging`: Log changes
  - `refactor`: Refactoring (no functional changes, no api changes)

### Flags
```
- [ ] Breaks backward compatibility
- [ ] Requires a documentation update
- [ ] Has untestable code
```
---

## Checklist
```
- [ ] Followed the guidelines in the CONTRIBUTING document
- [ ] Added prefix `[skip ci]`/`[ci skip]`/`[no ci]`/`[skip actions]`/`[actions skip]` in the PR title if necessary
- [ ] Tested and linted the code
- [ ] Commented the code
- [ ] Made corresponding changes to the documentation
```

## Checklist for maintainer
```
- [ ] Use `Squash and merge`
- [ ] Double-confirm the merge message has prefix `[skip ci]`/`[ci skip]`/`[no ci]`/`[skip actions]`/`[actions skip]`
- [ ] Delete the branch after merge
```

### CODE REVIEW

File: pkg/config/default.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ import (
 
 var (
 	// default values for X.509 certificate signing request
-	DEFAULT_COUNTRY             = "US"
+	DEFAULT_COUNTRY             = ""
```
[the remark this answers is not in the document]
Comment (t4niwa) [reply pkg/config/default.go:25]: 検証用PRなので後方互換は担保していないです。実際にマージする場合は、デフォルト値を修正しないようにする必要があると思います。

File: pkg/certificate/identity.go [hunk 2]
```diff
@@ -371,12 +371,28 @@ func PrepareIdentityCsrOptions(idCfg *config.IdentityConfig, domain, service str
...
+		Organization: func() []string {
+			if config.DEFAULT_ORGANIZATION != "" {
+				return []string{config.DEFAULT_ORGANIZATION}
+			}
+			return nil
+		}(),
 		OrganizationalUnit: []string{idCfg.ServiceCert.CopperArgos.Provider},
 		CommonName:         fmt.Sprintf("%s.%s", domain, service),
 	}
+	fmt.Printf("%#v", subject)
```
[the remark this answers is not in the document]
Comment (t4niwa) [reply pkg/certificate/identity.go]: 削除しました。

File: pkg/config/default.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (t4niwa) [reply pkg/config/default.go]: fixed in https://github.com/AthenZ/k8s-athenz-sia/pull/166/commits/62a5d030397d78e7b3b7133f270747f1bf98640b

**Review by y-myajima — APPROVED** [review]
LGTM

**Pull request merged**